packages/components/src/visually-hidden/style.scss

```diff
@@ -1,30 +0,0 @@
-.components-visually-hidden {
```

Comment:
Removing all these styles could potentially impact some third-party plugins that rely on them. The good news is that these are not official APIs, meaning plugins should be using components and not classnames. I think once we make more progress into merging G2 components and removing these old styles, we should consider writing a dev note to document the removed classnames.